### ISSUE TITLE
[BUG] - Can it support M4A format music

### ISSUE THREAD

**Issue by Yumipk** [CappielloAntonio/tempo#270]
M4A format music playback has no sound

**Comment by felipemarinho97** [comment]
m4a is just fine for me.. I'm using navidrome as the server

**Comment by DrAxelbauer** [comment]
I think I have the same issue. m4a files start to play and it seems, that the player is doing the playback but there is no sound output. Verified that the m4a is not damaged. 
I have attached the app log.
Would be cool, to find the problem or give a workaround.

Besides: Thanks for this massively cool app! Use it in combination with my nextcloud and it is what i was searching for for years!

[app_log.log](https://github.com/user-attachments/files/17274390/app_log.log)

**Comment by tofuliang** [comment]
Same issue, all Apple Lossless(m4a) failed to play.

**Comment by tofuliang** [comment]
Perhaps this issue is related to ExoPlayer? The author of Symfonium mentioned in a similar [issue](https://github.com/google/ExoPlayer/issues/6837).

**Comment by tofuliang** [comment]
@Yumipk @DrAxelbauer I built it with ffmpegAudioRenderer, and it now plays ALAC files well.
Maybe have a check [here](https://github.com/tofuliang/tempo/commit/66c26dee96465e3166ae0867a7e012f80cceeed4) and [here](https://github.com/tofuliang/tempo/releases)

**Comment by addy419** [comment]
Thanks @tofuliang, your solution works. Can it be added upstream?

**Comment by tofuliang** [comment]
> Thanks [@tofuliang](https://github.com/tofuliang), your solution works. Can it be added upstream?

maybe [4ec151906363d44dca9a7dd40199a3d6c78cedca](https://github.com/CappielloAntonio/tempo/commit/4ec151906363d44dca9a7dd40199a3d6c78cedca) did the same thing?

**Comment by addy419** [comment]
You're right! I didn't realise the F-Droid version was outdated. Updated to latest and it works now :)
